src/docker/tss/start.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+dockerd &
+
+sleep 10
+
+export DOCKER_DEFAULT_PLATFORM=linux/amd64
+
+poetry run python indexer_grpc_local.py start
+
+aptos init \
```

Comment:
I haven't actually run this bit yet but it's taken straight from /docker/chain (along with other bits) so it should work.